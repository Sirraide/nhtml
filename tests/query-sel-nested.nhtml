div {
    .test
    div {
        .test
    }
}

eval {
    $('div div .test').id = 'foobar'
    for (const el of $$('div .test')) el.classes += ' selected'
}