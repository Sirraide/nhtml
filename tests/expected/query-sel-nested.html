<div>
    <div class='selected test'></div>
    <div>
        <div class='selected test' id='foobar'></div>
    </div>
</div>
